### DISCUSSION TITLE
Drifting drone

### DISCUSSION THREAD

**Discussion by ch-stoyanov** [USC-ACTLab/crazyswarm#730] [Q&A]
We are trying to set up the Crazyswarm environment using Optitrack. We closely followed the instructions from the official forum (including flashing the recommended versions for the drone and the radio). The problem is that whenever we launch hover_swarm, the drone looks like it starts moving away on the simulation, even though in reality it is not moving (we recorded the behavior in the video below). Also, the if we run mocap, we get the correct position of the drone. Could you please give us some guidance on how to solve this? We also attached the config files, a screenshot of the mocap's output (which seems to show the correct position) and the output of the topic /cf6/pose (which seems to show the incorrect continuous moving of the drone).

https://youtu.be/AIzK9IGZDNE
![Screenshot from 2023-03-14 15-34-52](https://user-images.githubusercontent.com/40367225/225053854-fa073f1c-d665-4234-8606-cc33faeb3c3e.png)
![Screenshot from 2023-03-14 15-36-00](https://user-images.githubusercontent.com/40367225/225054187-36e3556d-6bd6-4286-a691-34ddbd9e4beb.png)
![Screenshot from 2023-03-14 15-36-50](https://user-images.githubusercontent.com/40367225/225054534-29b5725f-2517-4be6-985b-be95e27d8001.png)

**Comment by whoenig** [comment]
If you are starting out, please consider Crazyswarm2 instead (see https://imrclab.github.io/crazyswarm2/). 

cf6/pose is from the onboard estimate. The drift indicates that the motion capture data is not sent correctly, either because the radio runs an old firmware or because your object tracker settings are not updated for single-marker tracking.
